src/code/code_801780F0.c

```diff
@@ -0,0 +1,352 @@
+/**
+ * @file code_801780F0.c
```

Comment:
Any thoughts on a name for the file?

Reply:
In theory it lies between `speed_meter` and `sys_cfb`, so it might be something like `speed_mtx` or something. Name should probably mention mtx?

Reply:
`speed_mtx` seems fine to me.